cow_py/contracts/order.py

```diff
@@ -0,0 +1,243 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Literal, Optional, Union
+
+from eth_account.messages import _hash_eip191_message, encode_typed_data
+from eth_typing import Hash32, HexStr
+from eth_utils.conversions import to_bytes, to_hex
+
+
+@dataclass
+class Order:
+    # Sell token address.
+    sellToken: str
+    # Buy token address.
+    buyToken: str
+    # An optional address to receive the proceeds of the trade instead of the
+    # owner (i.e. the order signer).
+    receiver: str
+    # The order sell amount.
```

Comment:
Can we also use snake case here? On Pydantic you can define an `alias` so that when you export to JSON (or dict) the alias is used, I believe Dataclasses don't have that.
If too much work, neglect this comment.